src/v2/ion/responseTransformer.js

```diff
@@ -59,7 +59,9 @@ const getFirstLevelObjects = (resp) => {
 
 const getRemediationValues = (idx) => {
   const remediationValues = [];
-  if (_.isEmpty(idx.neededToProceed)) {
+  const hasSkipRemediationOnly =
```

Comment:
+1

Reply:
yes, we need that comment too, and also we need to update the comments inside the `if` block, since all those assume `neededToProceed` was empty, but that is no longer the case.
Or another way to do this is to have another `if` for when we only have `skip`, instead of mixing them with the cases when we have an empty `neededToProceed`

Reply:
Makes sense.. Let me look at this again

Reply:
I think it makes sense where it is. We currently don't have a form for the skip remediation, it shows a default view. We need to check the presence of messages anyway to get to the terminal view. This is why i think saying if either remediation is empty (neededToProceed is empty) or skip is the only remediation, we should handle it the same. Will update comments in the code. cc @haishengwu-okta